meshtastic/mesh_interface_runtime/send_pipeline.py

```
@@ -0,0 +1,854 @@
+            raise self._interface.MeshInterfaceError(
+                "Timed out waiting for interface config"
+            )
+
+    def waitForAckNak(self) -> None:
+        """Wait until an acknowledgement (ACK) or negative acknowledgement (NAK) is received or the wait times out."""
+        success = self._timeout.waitForAckNak(self._acknowledgment)
+        self._raise_wait_error_if_present(WAIT_ATTR_NAK)
+        if not success:
+            raise self._interface.MeshInterfaceError(
+                "Timed out waiting for an acknowledgment"
+            )
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)